### PR TITLE
[RHACS] ROX-19674: Documenting the need for private endpoint for ECR integration

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.3+

Issue:
https://issues.redhat.com/browse/ROX-19674

Link to docs preview:
https://75782--ocpdocs-pr.netlify.app/openshift-acs/latest/integration/integrate-with-image-registries.html

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-main` and cherry-pick to:
    - `rhacs-docs-4.5`
    - `rhacs-docs-4.4`
    - `rhacs-docs-4.3`

### CODE REVIEW

**Comment by srcporter** [comment]
lgtm!

**Review by srcporter — APPROVED** [review]
LGTM!

File: modules/manual-configuration-image-registry-ecr.adoc [hunk 1]
```diff
@@ -31,7 +31,7 @@ To erase changes you made to an automatically-generated ECR integration, delete
 .. *Update stored credentials*: Clear this box if you are modifying an integration without updating the credentials such as access keys and passwords.
 .. *Integration name*: The name of the integration.
 .. *Registry ID*: The ID of the registry.
-.. *Endpoint*: The address of the registry. This field is not enabled when the AssumeRole option is selected.
+.. *Endpoint*: The address of the registry. This value is required only if you are using a private VPC endpoint for Amazon ECR. This field is not enabled when the AssumeRole option is selected.
```
Comment (kcarmichael08) [inline modules/manual-configuration-image-registry-ecr.adoc:34]:
```suggestion
.. *Endpoint*: The address of the registry. This value is required only if you are using a private virtual private cloud (VPC) endpoint for Amazon ECR. This field is not enabled when the AssumeRole option is selected.
```

**Review by kcarmichael08 — COMMENTED** [review]
One minor suggestion; otherwise lgtm!

**Comment by kcarmichael08** [comment]
Needs a rebase also before merging.

**Pull request merged**